pkg/scaling/scale_handler.go

```
@@ -27,13 +27,21 @@
+const (
+	ScaleUp   ScaleDirection = "scale-up"
+	ScaleDown ScaleDirection = "scale-down"
+)
+
 type ScaleHandler struct {
 	kClient        *kubernetes.Clientset
 	kDynamicClient *dynamic.DynamicClient
 
 	scaleLocks sync.Map
 
 	logger *zap.Logger
```

golangci-lint (1.64.8)

File is not properly formatted

(gofmt)